src/app/components/mainContent/mainContent.component.tsx

```diff
@@ -19,28 +22,32 @@ export function MainContent(): JSX.Element {
     }
 
     return (
-        <div className="main-content-container">
-            <div className="category-items">
-                <CategoryItems
-                    categoryItems={content.articles?.slice(0, 2)}
-                    label={ContentLabel.Article}
-                    length={120}
-                />
-            </div>
-            <div className="category-items">
-                <CategoryItems
-                    categoryItems={content.reports?.slice(0, 2)}
-                    label={ContentLabel.Report}
-                    length={120}
-                />
-            </div>
-            <div className="category-items">
-                <CategoryItems
-                    categoryItems={content.blogs?.slice(0, 2)}
-                    label={ContentLabel.Blog}
-                    length={120}
-                />
+        <Fragment>
+            <LandingPage />
```

Comment:
it should be reversed, MainContent should be part of LandingPage, but leave it as it is for now